pkg/actor/deploy.go

```diff
@@ -46,7 +47,7 @@ func (d deploy) Handles(conds []api.ClusterCondition) bool {
 
 func (d deploy) Act(ctx context.Context, cluster *resource.Cluster) error {
 	log := d.log.WithValues("CrdbCluster", cluster.ObjectKey())
-	log.Info("reconciling resources")
+	log.V(int(zapcore.InfoLevel)).Info("reconciling resources")
```

Comment:
Same debug comment as above.  We may want to make this message a bit more informative as well?